prisonstaffhub-specs/src/test/resources/offenders-for-appointments.csv

```diff
@@ -0,0 +1,3 @@
+offenderNo,start time (example: 22:00 - 22:05 - 22:10 - 22:15 - 22:20 - 22:25 - 22:30 - 22:35 - 22:40 - 22:45 - 22:50 - 22:55)
+A12345,14:20
```

Comment:
Does this mean that the following are valid?

A12345, 22:00 - 22:05 
B12345,22:00-22:05

Why ignore the CSV format?
eg
offenderNo, startTime,endTime
A12345,22:00
A12345,22:00,22:05